content/security.md

```diff
@@ -184,14 +184,17 @@ const LISTS_METHODS = _.pluck([
 ], 'name');
 
 // Only allow 5 list operations per connection per second
-DDPRateLimiter.addRule({
-  name(name) {
-    return _.contains(LISTS_METHODS, name);
-  },
-
-  // Rate limit per connection ID
-  connectionId() { return true; }
-}, 5, 1000);
+
+if (Meteor.isServer) {
+  DDPRateLimiter.addRule({
+    name(name) {
+      return _.contains(LISTS_METHODS, name);
+    },
+
+    // Rate limit per connection ID
+    connectionId() { return true; }
+  }, 5, 1000);
+ }
```

Comment:
Thanks for submitting this.  Can you fix the indentation of that last parenthesis so it matches the opening `if` statement?